### PR TITLE
flac123: init at 0.0.12

### PR DESCRIPTION
**Pull request by lukateras**
###### Motivation for this change

`flac123` is a command-line FLAC player. I want to use it to play music. It is available as an installable package in many mainstream Unix-like operating systems:

Arch: https://www.archlinux.org/packages/community/x86_64/flac123/
Gentoo: https://packages.gentoo.org/packages/media-sound/flac123
FreeBSD: https://svnweb.freebsd.org/ports/head/audio/flac123/
OpenBSD: http://ports.su/audio/flac123

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] Linux
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/flac123/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    autoconf
+    automake -a
+  '';
+
+  buildInputs = [ autoconf automake flac libao libogg popt ];
+
+  meta = {
+    homepage = http://flac-tools.sourceforge.net/;
+    description = "A command-line program for playing FLAC audio files";
+    platforms = stdenv.lib.platforms.all;
```
Comment (joachifm) [inline pkgs/applications/audio/flac123/default.nix]: Please specify a license as well.

Comment (yegortimoshenko) [inline pkgs/applications/audio/flac123/default.nix]: @joachifm Done. Thanks!

**Comment by fpletz** [comment]
Added the package to all-packages.nix and used autoreconfHook instead of manually invoking autotools.

**Comment by yegortimoshenko** [comment]
@fpletz Thanks a lot! Will do that next time :-)

**Comment by fpletz** [comment]
Sure, no problem. It was IMHO easier to just fix those small nitpicks myself than another round of feedback-fix ping-pong. :)

**Pull request merged**